ggshield/utils.py

```diff
@@ -27,6 +28,7 @@
 
 EMPTY_SHA = "0000000000000000000000000000000000000000"
 EMPTY_TREE = "4b825dc642cb6eb9a060e54bf8d69288fbee4904"
+PRERECEIVE_TIMEOUT = 4.5
```

Comment:
it looks like a magic number, can you add a little comment about it please? 
not related to your PR but same from the one above, I was wondering where this SHA comes from